lib/Header/Header.less

```diff
@@ -125,24 +125,57 @@
 }
 
 // enyo-locale-non-latin
-.enyo-locale-non-latin .moon-header {
-	.moon-header-title, {
-		line-height: @moon-non-latin-header-text-line-height;
+.enyo-locale-non-latin {
+
+	// Languages with combining accent characters
+	&.enyo-locale-th, // Thai - Test Chars: ฟิ้ไััุุ
+	&.enyo-locale-ar, // Arabic
+	&.enyo-locale-fa, // Farsi
+	&.enyo-locale-ur, // Urdu
+	&.enyo-locale-ku, // Kurdish
+	&.enyo-locale-he, // Hebrew
+	&.enyo-locale-hi, // Hindi
+	&.enyo-locale-ta, // Tamil
+	&.enyo-locale-te, // Telugu
+	&.enyo-locale-kn, // Kannada
+	&.enyo-locale-ml, // Malayalam
+	&.enyo-locale-mr, // Marathi
+	&.enyo-locale-bn, // Bengali
+	&.enyo-locale-pa {// Panjabi
+		.moon-header .moon-header-title {
+			height: 159px;
+		}
 	}
-	&.moon-medium-header .moon-header-title, {
-		line-height: @moon-non-latin-medium-header-line-height;
+
+	/* Special override for Japanese font (LG Display_JP) which, as of 07-Feb-2014
+	has a taller baseline. This corrects both Header and InputHeader. 2014-02-24
+	Please remove the line-height rule below if this font file is changed. */
+	&.enyo-locale-ja .moon-header-title,
+	&.enyo-locale-ja .moon-input-header .moon-input-header-input-decorator > .moon-input {
+		line-height: 1.25em;
```

Comment:
Not sure - I thought you wrote that comment. :smile: No objections to removing it based on what you said.